content/300-guides/100-performance-and-optimization/150-connection-management/200-configure-pg-bouncer.mdx

```diff
@@ -41,7 +41,18 @@ Error: undefined: Database error
 Error querying the database: db error: ERROR: prepared statement "s0" already exists
 ```
 
-To work around this issue, you must connect directly to the database rather than going through PgBouncer. How to achieve this depends on your setup or provider:
+To work around this issue, you must connect directly to the database rather than going through PgBouncer. To achieve this, you can use the [`directUrl`](/reference/api-reference/prisma-schema-reference#datasource:~:text=the%20shadow%20database.-,directUrl,-No) field in your [`datasource`](/reference/api-reference/prisma-schema-reference#datasource) <span class="api"></span> block.
 
-- [Connecting directly to a PostgreSQL database hosted on Digital Ocean](https://github.com/prisma/prisma/issues/6157)
-- [Connecting directly to a PostgreSQL database hosted on ScaleGrid](https://github.com/prisma/prisma/issues/6701#issuecomment-824387959)
+For example, consider the following `datasource` block:
+
+```prisma
+datasource db {
+  provider  = "postgresql"
+  url       = "postgres://USER:PASSWORD@HOST:PORT/DATABASE?pgbouncer=true"
+  directUrl = "postgres://USER:PASSWORD@HOST:PORT/DATABASE"
```

Comment:
I recommend using environment variables instead of hard coding the connection string details here.

Reply:
Generally I agree, but that would require to also show the environment variable definition e.g. in `.env` to make sure people understand that the pgbouncer one should be in `url` - and I am not sure if this would make it to complicated for users to understand.

Any idea how to achieve that?

Reply:
The TabbedContent mdx component could be useful